package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "helpshift-docs",
+  "version": "0.0.0",
```

Comment:
When are we going to update the version? Do we have automation to increase the version with every release? Or is that something we are planning to do?

Reply:
We don't have an automation yet for bumping the version. We could use something like this: https://github.com/marketplace/actions/bumpr-bump-version-when-merging-pull-request-with-specific-labels, which is a Github Action to autobump the version under specific conditions. I'll explore this further.

Reply:
Please add a backlog task for this and resolve this comment.

Reply:
Done